src/main/java/core/basesyntax/service/UserValidator.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exeptions.RegistrationException;
+import core.basesyntax.model.User;
+
+public class UserValidator {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_PASSWORD_LENGTH = 6;
+    private static final int MIN_LOGIN_LENGTH = 6;
+
+    private final StorageDaoImpl storageDao;
+
+    public UserValidator() {
+        storageDao = new StorageDaoImpl();
+    }
+
+    public void validateAge(User user) throws RegistrationException {
```

Comment:
Unchecked exception should nt be used with `throws`
```suggestion
    public void validateAge(User user) {
```